src/fetcher.js

```diff
@@ -12,10 +12,10 @@ function fetchFile (batch, deliveryDate, config) {
     console.log(`fetching file from zuora with id ${batch.fileId}`)
     const options = {
       method: 'GET',
-      uri: `${config.url}/apps/api/batch-query/file/${batch.fileId}`,
+      uri: `${config.zuora.apiurl}/apps/api/batch-query/file/${batch.fileId}`,
```

Comment:
shouldn't this be config.zuora.api.url instead of config.zuora.apiurl ? and the same for all the other times we read the config